quint/test/types/constraintSolver.test.ts

```diff
@@ -192,6 +234,33 @@ describe('unify', () => {
     result.map(subs => assert.deepEqual(substitutionsToString(subs), '[ a |-> int, b |-> bool ]'))
   })
 
+  it('unifies sum-type', () => {
+    // We do not provide
```

Comment:
Seems like there is something missing in this comment

Reply:
Yep. But I don't know what. I've just removed it.